packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -932,10 +932,12 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
       }
       final RenderBox childRenderBox = _childKey.currentContext!.findRenderObject()! as RenderBox;
       final Size childSize = childRenderBox.size;
-      final double scaleChange = 1.0 - event.scrollDelta.dy / childSize.height;
-      if (scaleChange == 0.0) {
+      //Ignore left and right scroll
```

Comment:
We put a space after the `//` and end comments with a period.  See the [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) for more.

Same for the other comment below.